public/scripts/register.js

```diff
@@ -0,0 +1,21 @@
+const firstName = document.getElementById('first-name');
+const lastName = document.getElementById('last-name');
+const questTitle = document.getElementById('quest-title');
+const date = document.getElementById('date');
+const link = document.getElementById('link');
+const submitBtn = document.getElementById('submit-btn');
+
+const database = firebase.database();
+const rootRef = database.ref('/finishers');
+
+submitBtn.addEventListener('click', (e) => {
```

Comment:
you can just do `('click', e => {`

Reply:
or you can do `form.onsubmit = function (e) { ... }`  or  `form.onsubmit = e => { ... }` or  `submitBtn.onclick= e => { ... }` or `submitBtn.onclick= function(e) { ... }`